docker/dev/build.sh

```diff
@@ -1,14 +1,42 @@
 #!/bin/bash
+# Before run of this script you can set environmental variables
```

Comment:
Actually some prefer not run it this way and only set these variables for single command execution, like this:

```sh
$ BASE_IMG=ubuntu:18.04 ./build.sh
```

Reply:
ok